src/modal-manager/ModalManager.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import AbstractModal from 'terra-abstract-modal';
+import ActionHeader from 'terra-action-header';
+import SlideGroup from 'terra-slide-group';
+import ContentContainer from 'terra-content-container';
+
+import DisclosureManager, { availableDisclosureSizes } from '../disclosure-manager';
+import { navigationPromptResolutionOptionsShape } from '../navigation-prompt';
+
+import styles from './ModalManager.module.scss';
+
+const disclosureType = 'modal';
+export { disclosureType };
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The components to be rendered in the body of the ModalManager. These components will receive the
+   * disclosure capabilities through the DisclosureManger's context API.
+   */
+  children: PropTypes.node,
+  /**
+   * The component to render within the Modal above the disclosed content.
+   */
+  disclosureAccessory: PropTypes.element,
+  /**
+   * The Object (or function that returns an Object) that specifies the messages
+   * used to prompt the user when disclosure dismissal occurs when pending state
+   * is present.
+   *
+   * This prop will be fixed upon the initial mount of the component.
```

Comment:
I don't think this comment is accurate, if a consumer updated the options, this would get passed through and updated because the useEffect handles if this prop changes:
https://github.com/cerner/terra-application/pull/2/files#diff-4de323c09ec8dd1f9bee39d9703e0872R37

Reply:
It's accurate because DisclosureManager caches its disclosed components: https://github.com/cerner/terra-application/blob/updates-for-orion-application/src/disclosure-manager/DisclosureManager.jsx#L497

The DisclosureContainer is going to be created once with the current prop value and not updated again. So the comment is kind of accurate? Though an updated prop would apply to any subsequent disclosures...

Reply:
I could store the initial prop value in a ref and make things consistent throughout, but that seems a little heavy-handed. I could flex out this comment a bit more to describe the potential inconsistency and best practices for use.

Reply:
This is has been resolved by later changes 👍 The updated prop value should propagate through appropriately.